src/app/shared/services/tags/account-tag.service.ts

```diff
@@ -47,7 +47,7 @@ export class AccountTagService implements EntityTagService {
     return `${this.keys.sshDescription}.${sshKey.fingerprint}`;
   }
 
-  public writeTag(key: string, value: string): Observable<string> {
+  public writeTag(key: string, value: string): Observable<any> {
```

Comment:
why any?

Reply:
@zolotyx  I replaced string with any, because TagService.writeTag returns `TagService.update(entity: any, entityName: string, key: string, value: any): Observable<any>`. This method is normally used in different methods, which return Observables of different objects, not only strings.

Reply:
@ksendart , but tag service always returns string values of the tag. Or not?

Reply:
@zolotyx  TagService.update accept `entity: any` returns newEntity - object with new `tags` attribute.